src/registrar.ts

```diff
@@ -96,17 +96,23 @@ export function handleTransfer(event: Transfer): void {
   registrarContract.save();
 
   let domainId = toPaddedHexString(event.params.tokenId);
-  let domain = Domain.load(domainId);
-  if (domain === null) {
-    domain = new Domain(domainId);
-    domain.isLocked = false;
-    domain.royaltyAmount = BigInt.fromI32(0);
-  }
-  domain.owner = account.id;
-  if (domain.contract === null) {
-    domain.contract = getDefaultRegistrarForNetwork().toHexString();
+
+  if (account.id.localeCompare(ADDRESS_ZERO) === 0) {
```

Comment:
If we can just use `account.id === ADDRESS_ZERO` I think it would be simpler just to avoid using a locale function unnecessarily, but otherwise looks good

Reply:
It has weird thing, these two variables have same types and values, but strict equality still fails,  only `localCompare` returns that they are same.
The another way to compare address is to use `address.equal(Address.zero())`.